src/game.cpp

```diff
@@ -8917,7 +8919,8 @@ bool game::plfire_check( const targeting_data &args )
         }
 
         if( gun->has_flag( "MOUNTED_GUN" ) ) {
-            const bool v_mountable = static_cast<bool>( m.veh_at( u.pos() ).part_with_feature( "MOUNTABLE" ) );
+            const bool v_mountable = static_cast<bool>( m.veh_at( u.pos() ).part_with_feature( "MOUNTABLE",
+                                     true ) );
```

Comment:
Nitpick on this and the one below with the `true` var hanging off the newline, can I encourage:
```
const bool v_mountable =
    static_cast<bool>( m.veh_at( u.pos() ). part_with_feature( "MOUNTABLE", true ) )
```

Reply:
Anyway, that line is going to change again later: I plan on removing the boolean parameter and change this to calla a function `part_with_feature_including_broken`, or similar. Maybe even just a `cargo_part` function without parameters (that returns broken parts).